api/v1/openapi.yaml

```diff
@@ -960,6 +975,13 @@ definitions:
         "$ref": "#/definitions/Labels"
       to:
         "$ref": "#/definitions/Labels"
+      dports:
+        description: |
+          List of Layer 4 port and protocol pairs which the identity will use for
```

Comment:
> I think we should emphasize the "destination" because dports is being checked on both ingress and egress maps of the L4Policy in the destination label.

I'm not sure I understand your proposed sentence correctly. What is "the destination of the communication"? If it really does that then this is completely wrong. The dport must be validated against the egress policy of the source identity and in the ingress policy of the destination identity.